### ISSUE TITLE
fix

### ISSUE THREAD

**Issue by meiqicheng** [google-research/bert#1358]
原始的代码：
`    for index, (*box, conf, cls) in enumerate(o.detach().cpu().numpy()):
        targets.append([i, cls, *list(*xyxy2xywh(np.array(box)[None])),
                        conf, *list(kpts.detach().cpu().numpy()[index])])
`
修改后的代码
`    for index, (*box, conf, cls) in enumerate(o.detach().cpu().numpy()):
        # box格式为xyxy,而不是cxywh,不需要执行转换函数xyxy2xywh()
        targets.append([i, cls, *list(box),
                        conf, *list(kpts.detach().cpu().numpy()[index])])
`
